documentation/en/default-lotus-config.toml

```diff
@@ -330,6 +330,13 @@
   # env var: LOTUS_FEVM_ENABLEETHRPC
   #EnableEthRPC = false
 
+  # EnableActorEventsAPI enables the Actor events API that enables clients to consume events emitted by (smart contracts + built-in Actors).
+  # This will also enable the RealTimeFilterAPI and HistoricFilterAPI by default, but they can be disabled by config options above.
+  #
+  # type: bool
+  # env var: LOTUS_FEVM_ENABLEACTOREVENTSAPI
+  #EnableActorEventsAPI = false
```

Comment:
Discussion started but not resolved in https://github.com/filecoin-project/lotus/pull/11540#pullrequestreview-1841106215

We currently have a `Fevm` section in config, not `Fvm`. Do you think this warrants starting a new config section for one item?

Reply:
I think maybe an `Events` top level config option would be appropriate but that logically should move all of the current `Fevm.Events` options into it as well since this feature builds on top of it. But that probably needs a config section deprecation and duplicate handling during a deprecation period. I can do that if it makes sense but it seems heavy-handed.

Maybe the alternative is to put this in a new `Events` section alone for now and do a move of the other options at a later date as a cleanup.

Reply:
I agree with a top-level `Events` config option, although I'd keep the ETH-specific options where they are (although... maybe they all generalize?).

Reply:
> Maybe the alternative is to put this in a new Events section alone for now and do a move of the other options at a later date as a cleanup.

Yeah, this sounds like a good idea.

Reply:
Events top level config makes sense. make sure we have decoupled eth api enablement vs. native event subscription enablement, while keeping the coupling between eth api enablement & FEVM event enablement